app/services/bgs/dependent_service.rb

```diff
@@ -81,32 +81,38 @@ def submit_686c_form(claim)
         # why I am deliberately raising these errors here.
         validate_file_number_format!(file_number:)
         validate_file_number_matches_ssn!(file_number:)
-        if claim.submittable_686?
-          if Flipper.enabled?(:dependents_encrypt_jobs)
-            BGS::SubmitForm686cEncryptedJob.perform_async(
-              uuid, @icn, claim.id, encrypted_vet_info
-            )
-          else
-            BGS::SubmitForm686cJob.perform_async(
-              uuid, @icn, claim.id, form_hash_686c
-            )
-          end
-        else
-          if Flipper.enabled?(:dependents_encrypt_jobs) # rubocop:disable Style/IfInsideElse
-            BGS::SubmitForm674EncryptedJob.perform_async(
-              uuid, @icn, claim.id, encrypted_vet_info
-            )
-          else
-            BGS::SubmitForm674Job.perform_async(
-              uuid, @icn, claim.id, form_hash_686c
-            )
-          end
-        end
+        submit_form_job_id = if claim.submittable_686?
+                               if Flipper.enabled?(:dependents_encrypt_jobs)
+                                 BGS::SubmitForm686cEncryptedJob.perform_async(
+                                   uuid, @icn, claim.id, encrypted_vet_info
+                                 )
+                               else
+                                 BGS::SubmitForm686cJob.perform_async(
+                                   uuid, @icn, claim.id, form_hash_686c
+                                 )
+                               end
+                             else
+                               if Flipper.enabled?(:dependents_encrypt_jobs) # rubocop:disable Style/IfInsideElse
+                                 BGS::SubmitForm674EncryptedJob.perform_async(
+                                   uuid, @icn, claim.id, encrypted_vet_info
+                                 )
+                               else
+                                 BGS::SubmitForm674Job.perform_async(
+                                   uuid, @icn, claim.id, form_hash_686c
+                                 )
+                               end
+                             end
         Rails.logger.info('BGS::DependentService succeeded!', { user_uuid: uuid, saved_claim_id: claim.id, icn: })
       end
+
+      {
+        submit_pdf_job_id:,
+        submit_form_job_id:
+      }
```

Comment:
having job ids returns is helpful for debugging and a good sanity check that an async job did get kicked off